modules/zip/src/parse-zip/zip-compozition.ts

```diff
@@ -1,8 +1,13 @@
 import {FileHandleFile, concatenateArrayBuffers} from '@loaders.gl/loader-utils';
-import {ZipEoCDRecord, parseEoCDRecord, updateEoCD} from './end-of-central-directory';
+import {ZipEoCDRecord, generateEoCD, parseEoCDRecord, updateEoCD} from './end-of-central-directory';
```

Comment:
Nit: Filename typo? Should be `zip-composition.ts`? (No "z")

Reply:
Renamed